js/localization/messages/sv.json

```diff
@@ -0,0 +1,264 @@
+        "dxDataGrid-filterRowOperationEquals": "Är lika med",
+        "dxDataGrid-filterRowOperationNotEquals": "Är olika",
+        "dxDataGrid-filterRowOperationLess": "Mindre än",
+        "dxDataGrid-filterRowOperationLessOrEquals": "Mindre än eller lika med",
+        "dxDataGrid-filterRowOperationGreater": "Större än",
+        "dxDataGrid-filterRowOperationGreaterOrEquals": "Större än eller lika med",
+        "dxDataGrid-filterRowOperationStartsWith": "Börjar med",
+        "dxDataGrid-filterRowOperationContains": "Innehåller",
+        "dxDataGrid-filterRowOperationNotContains": "Innehåller inte",
+        "dxDataGrid-filterRowOperationEndsWith": "Slutar med",
+        "dxDataGrid-filterRowOperationBetween": "Mellan",
+        "dxDataGrid-filterRowOperationBetweenStartText": "Start",
+        "dxDataGrid-filterRowOperationBetweenEndText": "Slut",
+        "dxDataGrid-applyFilterText": "Använd filter",
+        "dxDataGrid-trueText": "sant",
+        "dxDataGrid-falseText": "falskt",
+        "dxDataGrid-sortingAscendingText": "Sortera stigande",
+        "dxDataGrid-sortingDescendingText": "Sortera fallande",
+        "dxDataGrid-sortingClearText": "Rensa sortering",
+        "dxDataGrid-editingSaveAllChanges": "Spara ändringar",
+        "dxDataGrid-editingCancelAllChanges": "Ångra ändringar",
+        "dxDataGrid-editingAddRow": "Lägg till rad",
+        "dxDataGrid-summaryMin": "Min: {0}",
+        "dxDataGrid-summaryMinOtherColumn": "Minimi av {1} är {0}",
+        "dxDataGrid-summaryMax": "Max: {0}",
+        "dxDataGrid-summaryMaxOtherColumn": "Maximi av {1} är {0}",
+        "dxDataGrid-summaryAvg": "Medel: {0}",
+        "dxDataGrid-summaryAvgOtherColumn": "Medeltal av {1} är {0}",
+        "dxDataGrid-summarySum": "Sum: {0}",
+        "dxDataGrid-summarySumOtherColumn": "Summan av {1} är {0}",
+        "dxDataGrid-summaryCount": "Antal: {0}",
+        "dxDataGrid-columnFixingFix": "Fixera",
+        "dxDataGrid-columnFixingUnfix": "Avfixera",
+        "dxDataGrid-columnFixingLeftPosition": "Till vänster",
+        "dxDataGrid-columnFixingRightPosition": "Till höger",
+        "dxDataGrid-exportTo": "Exportera",
+        "dxDataGrid-exportToExcel": "Exportera till Excel fil",
+        "dxDataGrid-excelFormat": "Excel fil",
+        "dxDataGrid-selectedRows": "Valda rader",
+        "dxDataGrid-exportSelectedRows": "Exportera valda rader",
+        "dxDataGrid-exportAll": "Exportera allt",
+        "dxDataGrid-headerFilterEmptyValue": "(Tomma)",
+        "dxDataGrid-headerFilterOK": "OK",
+        "dxDataGrid-headerFilterCancel": "Avbryt",
+        "dxDataGrid-ariaColumn": "Kolumn",
+        "dxDataGrid-ariaValue": "Värde",
+        "dxDataGrid-ariaFilterCell": "Filtrera cell",
+        "dxDataGrid-ariaCollapse": "Kollapsa",
+        "dxDataGrid-ariaExpand": "Expandera",
+        "dxDataGrid-ariaDataGrid": "Datarutnät",
+        "dxDataGrid-ariaSearchInGrid": "Säk i datarutnätet",
```

Comment:
@aenges Is it the right translation for "search" or it should be "Sök"?

Reply:
Sorry, should be "Sök".